src/test/kotlin/blackjack/domain/model/DealerTest.kt

```diff
@@ -0,0 +1,47 @@
+package blackjack.domain.model
+
+import blackjack.domain.PointCalculator
+import io.kotest.matchers.shouldBe
+import org.junit.jupiter.api.Test
+
+class DealerTest {
+    @Test
+    fun `딜러의 이름은 딜러이다`() {
```

Comment:
```suggestion
    fun `딜러의 이름의 기본값은 딜러이다`() {
```
딜러의 이름을 딜러라고 한다면, 딜러의 이름은 변경하지 못하는 것처럼 오해할 수 있을 것 같아요.
요구사항은 언제든지 변경이 가능하니, 이 부분을 염두에 두었으면 좋겠습니다. 😄